.github/workflows/virtualbox_image.yml

```diff
@@ -0,0 +1,43 @@
+name: Create VirtualBox disk image from Dockerfile
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  push:
+    paths:
+      - 'docker/*'
+      - '.github/workflows/virtualbox_image.yml'
+
+env:
+  VM_PASSWORD: temppwd
+
+jobs:
+  virtualbox_image_build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        stage: [base_python, moab, dagmc, openmc]
+        hdf5: ['']
+        include:
+          - stage: dagmc
+            hdf5: _hdf5
+      fail-fast: false
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build VirtualBox image using d2vm
+        run: |
+          curl -sL "https://github.com/linka-cloud/d2vm/releases/download/v0.2.0/d2vm_v0.2.0_linux_amd64.tar.gz" | tar -xvz d2vm
+          sudo mv d2vm /usr/local/bin/
+          echo "make worked"
+          sudo d2vm convert ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}:latest -p ${{ env.VM_PASSWORD }} -o pyne.vdi
```

Comment:
This may work for the PyNE GHCR repo if we only pick the final image with PyNE installed??

Reply:
I've just looked at the packages of PyNE, `docker_publish.yml`, and the dockerfile and I realized that we don't ever build the `pyne` stage at the end of the dockerfile, which means that there are no packages of PyNE that have PyNE installed. We might have to merge a PR that changes the inputs to the multistage build action to include the `pyne` stage before we continue with with this PR.

Reply:
Right! I keep getting this confused with the DAGMC case.

Maybe the following?
- Add a stage to build PyNE in the Dockerfile 
- change this new workflow to be about deployment so it will:
   * build & pus the docker image with PyNE using the multistage build for efficiency/consistency 
   * convert it to a VM image

Reply:
I think we should add the building of the docker image with PyNE to the `docker_publish.yml` workflow because it already has all the infrastructure of the multistage build actions and the jobs that will test and push the images if they are working. I think the `virtualbox_image.yml` workflow should only take the PyNE docker image from ghcr.io and convert it into a VM image.

Reply:
But then don't we have to run that action with every merged pull-request? Maybe the docker image caching provided by the multistage merge action means that it's not too much load? We currently only run this action rarely to update the dependencies.